src/Build/Instance/ProjectItemInstance.cs

```diff
@@ -1695,11 +1697,21 @@ internal void TranslateWithInterning(ITranslator translator, LookasideStringInte
                     {
                         int count = translator.Reader.ReadInt32();
                         _directMetadata = (count == 0) ? null : new CopyOnWritePropertyDictionary<ProjectMetadataInstance>();
```

Comment:
Is this line redundant now?

Reply:
Yes, it is leftover. Good catch. Deleted.